googledatastore/src/main/java/com/yahoo/ycsb/db/package-info.java

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) 2015 Google Inc. All rights reserved.
```

Comment:
Same coment as above re: Google as named copyright rather than the general "YCSB contributors"